src/noisepy/seis/main.py

```diff
@@ -37,6 +38,11 @@ class Command(Enum):
     ALL = 4
 
 
+class DataFormat(Enum):
+    ZARR = 1
+    ASDF = 2
```

Comment:
Are you actually using the values of this enum anywhere? Would it be better to be the lower name values instead? Seems like that's the only thing you're using it for, might save some of the `.lower` computation.

Reply:
It's a habit since an int comparison is generally much faster than a string comparison and has fewer gotchas (like casing and culture). But I agree in this case the string value makes things simpler and won't make a difference perf-wise. I'll update ...